public/modules/custom/grants_metadata/tests/src/Kernel/AtvSchemaTest.php

```diff
@@ -611,7 +611,7 @@ public function testAttachments() : void {
     $applicationData->setValue($submissionData);
     // Search attachment data.
     $attachmentField = [];
-    foreach ($applicationData as $field) {
+    foreach ($submissionData as $field) {
       $name = $field->getName();
```

Comment:
Toi $field näyttäs olevan stringi ainakin joskus, koska yksikkötestit feilaa tuohon.

![image](https://github.com/City-of-Helsinki/hel-fi-drupal-grants/assets/892920/14a0864e-9113-4a38-af50-cc1d80a464f3)

![image](https://github.com/City-of-Helsinki/hel-fi-drupal-grants/assets/892920/45fa5369-8065-477b-87b7-7a23ef179fa8)

Mä yritin jollain nopeella fiksillä, mut pitäs ottaa vähän syvempi daivi tohon, että pystys lähtee korjailee.

Ehkä @Puavo auttaa ton selvittämisessä? Että saatas noi yksikkötestitki läpi.

Reply:
Ehkä johtuu tästä muutoksesta, että tuolla on sisällä eri matskua?

![image](https://github.com/City-of-Helsinki/hel-fi-drupal-grants/assets/892920/b5509f4d-8b92-4953-8797-e3607af819ee)

Reply:
Minäpä tutkin tätä.

Reply:
Ehkä jatkossa tämän tyyppisiä voisi vaan laittaa phpstan-ignoreen ja kertoa kommentissa miksi se on tehty. Tein tähän pienimuotoisen rewriten joka selkeyttää koodia ja väistää staattisen analyysin keksimän epäongelman.

Reply:
Joo. Nyt menee yksikkötestitkin läpi